roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | replace(':','=') | replace('{', '') | replace('}', '') | replace(' ', '') | replace(\"u\'\", '') | replace(\"\'\", '') }}"
```

Comment:
@zgalor my mistake, it does work with multiple selectors in its current form.

When I mentioned dict_to_string i had something like this in mind in the filter_plugin

def selector_to_string(user_dict):
  selectors = []
  for key in user_dict:
    selectors.append("{}={}".format(key, user_dict[key]))
  return ",".join(selectors)